### ISSUE TITLE
Questions in pretrained speech translation

### ISSUE THREAD

**Issue by wang304381190** [espnet/espnet#3061]
I use pretrained ASR model for ST in EN->ZH. 
Firstly, I train an asr model until I get ctc loss equals 7. But when I fine-tune it on ST, the loss_asr equals 1762 (mtl_alpha=1) in first 100 iteration. Do I make a mistake in load asr model? 
I set             enc-init-mods: encoder.embed,encoder.encoders,encoder.after_norm

**Comment by wang304381190** [comment]
Thanks for your attention and apply so quickly, I'm sorry to bother you so late.
I also think the problem is in loading vocab, but I really don't know how to solve it.
Because Chinese and English is different absolutely, so I made dictionary respectively, 10k bpe for Chinese and 5k bpe for English.
I copied the dict_en in asr1/data/lang_1spm/train_sp_bpe5000_units.txt  to st1/data/lang_1spm/train_sp_bpe5000_units.txt directly, so it's same. And I use this dictionary to updata json for adding source sentences.
Also, I pass on dict_en as argument for st_train, and add this in error_calculator_asr.
What else could be wrong?
I'm looking forward to your reply.

**Comment by hirofumi0810** [comment]
@wang304381190 I think it's not a problem. It is natural to think that ASR and ST models use different vocabularies.
ASR vocab is usually built on transcription only. However, ST/MT vocabs are build on the joint transcription and translation.
You don't have to copy the source dict (En). Instead, you can build the joint En and Zh vocabulary in the ST recipe.
Although Kanji characters and alphabets are very different, this can simply the training pipeline.

Also, don't worry about high ASR loss values in the early ST training stage.

**Comment by wang304381190** [comment]
How about enc-init-mods? Only encoder.encoders is enough? Will the addition of encoder.embed and encoder.after_norm harm the performance?
Also, I would like to accomplish ST from English to Chinese, but the BLUE is terrible every time, could you please help me find the reasons? Training information are as follows.

I set train.conf as:   
mtlalpha: 1.0  asr-weight: 0.2  mt-weight: 0.2  enc-init-mods: encoder.embed,encoder.encoders,encoder.after_norm

loss:
![loss](https://user-images.githubusercontent.com/55871262/110934312-cf23cf00-8368-11eb-8246-fa57ea8e1369.png)


BLUE:
![bleu](https://user-images.githubusercontent.com/55871262/110934188-a8659880-8368-11eb-90a9-4a427bf5a28c.png)

acc:
![acc](https://user-images.githubusercontent.com/55871262/110934269-c16e4980-8368-11eb-8757-6445a6c8069d.png)

@hirofumi0810
